app/components/preprint-form-section.js

```diff
@@ -74,7 +74,7 @@ export default CpPanelComponent.extend(Analytics, {
                     });
                 this._super(...arguments);
             } else {
-                this.sendAction('errorAction', this.get('denyOpenMessage')); /* eslint-disable-line ember/closure-actions */
+                this.get('errorAction')(this.get('denyOpenMessage'));
```

Comment:
You don't need to use `.get()` here to get the closure action. You can just do:
```
this.errorAction(this.get('denyOpenMessage'));
```
Also, since `denyOpenMessage` is not a computed, you can also access it without `.get()`:
```
this.errorAction(this.denyOpenMessage);
```
(BTW, in Ember 3.1, you can even access computeds without `.get()`)